packages/application-components/src/components/modal-pages/internals/modal-page.styles.ts

```diff
@@ -1,26 +1,53 @@
 import { css, type SerializedStyles } from '@emotion/react';
 import { customProperties } from '@commercetools-uikit/design-system';
+import type { TModalPageSize } from './modal-page';
 
-export const TRANSITION_DURATION = 200;
+const TRANSITION_DURATION = 200;
+const LONGER_TRANSITION_DURATION = 300;
```

Comment:
One of the reasons why I suggested to use the styles mapper object was to get rid of these constants, as they don't seem so useful now.

However, if you prefer to keep them, I would suggest renaming them as they're not very self explanatory right now.
Maybe something like: 
* `TRANSITION_DURATION_SHORT` and `TRANSITION_DURATION_LONG`
* `FULL_WIDTH_TRANSITION_DURATION` and `NARROWER_WIDTH_TRANSLATION_DURATION` 😨

Reply:
Removed them.